docs/sandbox-permissions.rst

```diff
@@ -223,9 +223,26 @@ requested with ``--filesystem`` and are not available with
 
 Device access
 `````````````
+You can provide the following device permissions:
+
+========= ======================================================
+``dri``   Direct Rendering Interface. Necessary for GL.
+``kvm``   Kernel based Virtual Machine ``/dev/kvm``
+``shm``   Shared Memory in ``/dev/shm``.
+``input`` Input devices as exposed in ``/dev/input``. This includes game controllers. Since Flatpak 1.15.6.
+``all``   All devices, including all of the above except ``shm``
+========= ======================================================
+
+.. note::
+
+   ALSA sound devices in ``/dev/snd`` are exposed with the socket
+   permission ``pulseaudio``.
```

Comment:
We should mention the bits about mic access too. It comes up often. Effectively on systems that use pulseaudio this is recording, streaming, microphone and direct ALSA right now.

Reply:
and I'll add MIDI too.

true story: several bits of audio software I packaged have device=all because of MIDI (and ALSA).

removing them progressively.